adafruit_ble/uuid/__init__.py

```diff
@@ -49,6 +49,11 @@ def __eq__(self, other):
     def __str__(self):
         return str(self.bleio_uuid)
 
+    def __bytes__(self):
+        b = bytearray(self.bleio_uuid.size // 8)
+        self.bleio_uuid.pack_into(b)
+        return bytes(b)
+
```

Comment:
`_bleio.UUID.uuid128` already returns a `bytes`, so you only need to convert `.uuid16`